README.md

```diff
@@ -0,0 +1,59 @@
+neighborhood-noticing
+=====================
+
+# Architecture
+
+This is a front-end Javascript application built with Vue.js. It pulls the data it needs from Citygram via its API.
+
+For Citygram's architecture, see [this
+diagram](https://github.com/codeforamerica/citygram/blob/master/images/citygram_architecture.png).
+
+# Development
+
+* Install nodejs@10.6
+  * Use of [nvm](https://github.com/creationix/nvm) recommended
+* Install [`yarn`](https://yarnpkg.com)
+* `yarn install`
+* Copy environment file examples
+  * `cp .env.example .env`
+    * Load this using autoenv, direnv, or other
+  * `cp .citygram.env.example .citygram.env`
+  * `cp .citygram-services.env.example .citygram-services.env`
+    * Set `$SOCRATA_APP_TOKEN` in order to use endpoints that consume the Socrata API. See
+      [Socrata Application Tokens](https://dev.socrata.com/docs/app-tokens.html) for Socrata token creation instructions
+      and details. See [Socrata Applications](https://dev.socrata.com/register) to register a new application (and
+      receive a token).
+* Start dependencies
+  * `docker-compose up -d db`
+  * `docker-compose up citygram_migration citygram_services_migration`
+  * `docker-compose up`
+    * TODO: have migration tasks wait for database availability so this can be reduced to just `docker-compose up`
+* `npm start`
```

Comment:
i usually enjoy it if you specify which port the app will be served on. that's just me though. no pressure!

Reply:
👍 3f62f6b